app/src/main/java/mod/trindadedev/ui/fragments/events/EventsManagerDetailsFragment.java

```diff
@@ -0,0 +1,190 @@
+package mod.trindadedev.ui.fragments.events;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.gson.Gson;
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+
+import com.sketchware.remod.R;
+import com.sketchware.remod.databinding.FragmentEventsManagerDetailsBinding;
+import com.sketchware.remod.databinding.LayoutEventItemBinding;
+
+import mod.trindadedev.ui.fragments.BaseFragment;
+import mod.jbk.util.OldResourceIdMapper;
+import mod.agus.jcoderz.lib.FileUtil;
+import mod.hey.studios.util.Helper;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class EventsManagerDetailsFragment extends BaseFragment {
+     
+     private final ArrayList<HashMap<String, Object>> listMap = new ArrayList<>();
+     private String listName;
+     private FragmentEventsManagerDetailsBinding binding;
+     
+     public EventsManagerDetailsFragment () {
+          listName = "";
+     }
+     
+     public EventsManagerDetailsFragment(String listName) {
+          this.listName = listName;
+     }
+
+     @Override
+     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+          binding = FragmentEventsManagerDetailsBinding.inflate(inflater, container, false);
+          return binding.getRoot();
+     }
+
+     @Override
+     public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
+          super.onViewCreated(view, savedInstanceState);
+          configureToolbar(binding.toolbar);
+          binding.fabNewEvent.setOnClickListener(v -> {
+                Bundle args = new Bundle();
+                args.putString("lis_name", listName);
+                EventsManagerCreatorFragment fragment = new EventsManagerCreatorFragment();
+                fragment.setArguments(args);
+                openFragment(fragment);
```

Comment:
You can add `EventsManagerCreatorFragment#newInstance(String)` for better readability.